### PR TITLE
CANVAS-145 - Improved UI for Course builder.

### PR DESCRIPTION
**Pull request by tatikamsetti**
@rlorenzo Can you please review the code? 
tq.

### CODE REVIEW

File: src/client/CourseRequestForm/index.js [hunk 1]
```diff
@@ -223,18 +229,18 @@ const CourseRequestForm = ({ courses }) => {
     ))
   );
   const courseRequestFormHeaderList = [
-    'Request ID',
+    // 'Request ID',
```
Comment (rlorenzo) [inline src/client/CourseRequestForm/index.js:232]: @tatikamsetti Why is this commented out? For new requests, it should just be blank like it is currently on CCLE

File: src/client/CourseRequestForm/index.js [hunk 2]
```diff
@@ -85,21 +91,21 @@ const CourseRequestForm = ({ courses }) => {
     coursesData[index].toBeBuilt = !coursesData[index].toBeBuilt;
     setCoursesData(coursesData);
   };
-  const date = new Date();
-  // Const updatedDate = `${date.getFullYear()}-${}`;
+
+  const markedDate = moment(new Date()).format('YYYY-MM-DD h:mm A');
```
Comment (rlorenzo) [inline src/client/CourseRequestForm/index.js:95]: @tatikamsetti Reading up more on Moment.js and it seems the project is now defunct: https://momentjs.com/docs/#/-project-status/

Should be able to get similar results using:

Date.toLocaleDateString() and Date.toLocaleTimeString()

**Review by rlorenzo — CHANGES_REQUESTED** [review]

File: src/client/CourseRequestForm/index.js (hunk 1 above)
Comment (tatikamsetti) [reply src/client/CourseRequestForm/index.js:232]: @rlorenzo As the ticket says... it should match the screenshot attached and I do not see Request Id in the header list. Also to confirm, I looked at ccle's course requestor and deleted it.
See the CCLE's course requestor form:
Screen Shot 2020-12-02 at 6 34 15 PM

I can add back if you want me to.

File: src/client/CourseRequestForm/index.js (hunk 2 above)
Comment (tatikamsetti) [reply src/client/CourseRequestForm/index.js:95]: @rlorenzo Ok changed it.

**Pull request closed without merging**